tests/e2e-targetallocator/targetallocator-features/00-assert.yaml

```diff
@@ -20,7 +20,7 @@ spec:
           items:
           - key: collector.yaml
             path: collector.yaml
-          name: stateful-collector-85dbe673
+          name: stateful-collector-57be076a
```

Comment:
OOC why are the hashes changing? id be interested to understand how the full content is being changed.